packages/common/src/paginator.js

```diff
@@ -231,4 +247,25 @@ streamRouter.runAsStream_ = function(parsedArguments, originalMethod) {
   return limiter.stream;
 };
 
-module.exports = streamRouter;
+/**
+ * Wraps original method in auto paginated function and provides a getter
+ * for the original in the event that we wish to streamify that same method
+ * later.
+ *
+ * @param {function} originalMethod - The original method.
+ * @return {function} wrapped - The wrapped method.
+ */
+paginator.wrap_ = function(originalMethod) {
```

Comment:
@stephenplusplus this is the code I'd like some feedback around. I saw several options for how to separate the streaming logic, however I think the one with the least amount of overhead and most flexibility was to create a getter for the original method.